### DISCUSSION TITLE
Change font serenity template

### DISCUSSION THREAD

**Discussion by juancruzsosagab** [primefaces/community#838] [PrimeReact Templates] [answered]
hi guys, im currently using a theme(serenity) from primereact for my app, i want to change all the fonts, wich is the besty way? i dont understan how exactly i can do, because if using the layout light green and it has its own fonts for each component

**Comment by mertsincan** [comment]
@gucal can you assist with this question?

**Comment by buraksaglam089** [comment, marked as answer]
Hello,

To update the fonts in your project, please follow these steps:

First, add the desired fonts to your project's fonts folder.
Next, ensure that these new fonts are included in the _fonts.scss file.
After integrating the fonts into _fonts.scss, verify that they are correctly imported in the _layout.scss file.
Finally, to apply these changes, you need to compile the SASS command.
By following these steps, you should be able to change your fonts successfully.

Best regards,
